woocommerce/woocommerce
# Clean up unused images in `./assets/image` folder

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Remove the following unused images:

- assets/images/admin_notes/filter-by-product-variations-note.svg
- assets/images/empty-content.svg
- assets/images/icons/credit-cards/amex.png
- assets/images/icons/credit-cards/diners.png
- assets/images/icons/credit-cards/discover.png
- assets/images/icons/credit-cards/jcb.png
- assets/images/icons/credit-cards/laser.png
- assets/images/icons/credit-cards/maestro.png
- assets/images/icons/credit-cards/mastercard.png
- assets/images/icons/global-attributes-icon.svg
- assets/images/paypal-braintree.png
- assets/images/select2-spinner.gif
- assets/images/select2.png
- assets/images/select2x2.png
- assets/images/task_list/basics-section-illustration.png
- assets/images/wcpayments-icon-secure.png
- assets/images/wpspin.gif




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

If you’re using Visual Studio Code, you can use the global search feature:
Open VSCode and press Ctrl+Shift+F (or Cmd+Shift+F on Mac).
Type the image file name path in the search bar.

1. Check the images are not referenced in the codebase.




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
